[DOCS-1265]DOCS-1291]DOCS-1253][DOCS-1255] Docs updates for runs (#1144)
wandb/docs: content/guides/models/track/runs/grouping.md

Use `WANDB_RUN_GROUP` to specify a group for your runs as an environment variable. For more on this, check our docs for [**Environment Variables**]({{< relref "/guides/models/track/environment-variables.md" >}})**. Group** should be unique within your project and shared by all runs in the group. You can use `wandb.util.generate_id()` to generate a unique 8 character string to use in all your processes— for example, `os.environ["WANDB_RUN_GROUP"] = "experiment-" + wandb.util.generate_id()`

### 3. Set a group in the UI

After a run is initialized, you can move it to a new group from your workspace or its **Runs** page.

1. Navigate to your W&B project.
1. Select the **Workspace** or **Runs** tab from the project sidebar.
1. Search or scroll to the run you want to rename.

>     Hover over the run name, click the three vertical dots, then click **Move to another group**.

Reviewer: Missing number, extra line between list items.

```suggestion
    Hover over the run name, click the three vertical dots, then click **Move to another group**.
```

1. To create a new group, click **New group**. Type a group name, then submit the form.
1. Select the run's new group from the list, then click **Move**.

### 4. Toggle grouping by columns in the UI

You can dynamically group by any column, including a column that is hidden. For example, if you use `wandb.config` to log batch size or learning rate, you can then group by those hyperparameters dynamically in the web app. The **Group by** feature is distinct from a [run's run group]({{< relref "grouping.md" >}}). You can group runs by run group. To move a run to a different run group, refer to [Set a group in the UI]({{< relref "#set-a-group-in-the-ui" >}}).

> {{% alert title='optional_title' %}}

Reviewer: `title='optional_title'`  left in by mistake?

> {{% alert title='optional_title' %}}

Author: ```suggestion
{{% alert %}}
```

In the list of runs, the **Group** column is hidden by default.
{{% /alert %}}

To group runs by one or more columns:

1. Click **Group**.
1. Click the names of one or more columns.
1. If you selected more than one column, drag them to change the grouping order.
1. Click anywhere outside of the form to dismiss it.

### Customize how runs are displayed

> You can customize how runs are displayed in your workspace from the **Workspace** or **Runs** tab, which share display settings.

Reviewer: See comment above about this part sounding confusing 

"...which share display settings"

> You can customize how runs are displayed in your workspace from the **Workspace** or **Runs** tab, which share display settings.

Author: ```suggestion
You can customize how runs are displayed in your project from the **Workspace** or **Runs** tabs. Both tabs use the same display configuration.
```

To customize which columns are visible:

> 1. Above the list of runs, click **Columns**.

Reviewer: I think the numbering is rendering incorrectly. This is what I see, note the single "1." after "Drag columns to change order": 
![Uploading Screenshot 2025-04-08 at 3.04.46 PM.png…]()

Author: I'm puzzled. This is what I see locally: 
![Screenshot 2025-04-08 at 3 37 10 PM](https://github.com/user-attachments/assets/bc9dc010-c750-454d-a9be-26627dceb5e2)
Let's see if the PR updates now.

Use `WANDB_RUN_GROUP` to specify a group for your runs as an environment variable. For more on this, check our docs for [**Environment Variables**]({{< relref "/guides/models/track/environment-variables.md" >}})**. Group** should be unique within your project and shared by all runs in the group. You can use `wandb.util.generate_id()` to generate a unique 8 character string to use in all your processes— for example, `os.environ["WANDB_RUN_GROUP"] = "experiment-" + wandb.util.generate_id()`

### 3. Set a group in the UI

> To set a run's group in the UI:

Author: ```suggestion
```

After a run is initialized, you can move it to a new group from your workspace or its **Runs** page.

1. Navigate to your W&B project.
1. Select the **Workspace** or **Runs** tab from the project sidebar.
1. Search or scroll to the run you want to rename.

>   Hover over the run name, click the three vertical dots, then click **Move to another group**.

Author: ```suggestion
    Hover over the run name, click the three vertical dots, then click **Move to another group**.
```
